### ISSUE TITLE
AutoForm styling (payments)

### ISSUE THREAD

**Issue by aaronjudd** [reactioncommerce/reaction#1982]
Seen in #1974 

Some of the AutoForm inputs in the dashboard are missing styles. See payments.

<img width="369" alt="basic_reaction_product" src="https://cloud.githubusercontent.com/assets/439959/23926207/175e4ff6-08d0-11e7-8963-33f72bda2b36.png">

I think this was previously resolved with a global style patch that was applied elsewhere.. (but not by reattaching class="form-control").

**Comment by mikemurray** [comment]
This was me. I removed the following because I was messing up the styles of the React form fields. I'll add it back, but with some tweaks to the styles

```
.admin-controls-content .form-group input:not([type=checkbox]) {
    .form-control;
}
```

**Comment by aaronjudd** [comment]
that'd be nice.

**Comment by zenweasel** [comment]
Closed via #1998
